### PR TITLE
Enhancements for workers-scale

### PR DESCRIPTION
**Pull request by vishnuchalla**
## Type of change

- [ ] Refactor
- [ ] New feature
- [x] Bug fix
- [x] Optimization
- [ ] Documentation Update

## Description
Adding enhancements for workers scale and in for kube-burner in overall. Has all the changes incorporated in this PR: https://github.com/kube-burner/kube-burner-ocp/pull/136 as part of the release.

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
Will be verified through PR rehearsal jobs.

### CODE REVIEW

File: workloads/kube-burner-ocp-wrapper/run.sh [hunk 1]
```diff
@@ -124,7 +124,6 @@ download_binary
 if [[ ${WORKLOAD} =~ "index" || ${WORKLOAD} =~ "workers-scale" ]]; then
   if [[ "$START_TIME" != 0 && "$END_TIME" != 0 ]]; then
     JOB_START=$(date -u -d "@$START_TIME" +"%Y-%m-%dT%H:%M:%SZ")
-    END_TIME=$(expr $END_TIME + 600)
```
Comment (chentex) [inline workloads/kube-burner-ocp-wrapper/run.sh:127]: You removed this line, but the variable is used below.

Comment (vishnuchalla) [reply workloads/kube-burner-ocp-wrapper/run.sh:127]: Still its valid and the variable is present. We are just removing the 10 minutes add at the end.

**Review by dry923 — APPROVED** [review]
LGTM

**Pull request merged**